nixos/modules/services/networking/cntlm.nix

```
@@ -75,11 +78,13 @@ in
         A list of domains where the proxy is skipped.
       '';
       default = [];
+      type = types.listOf types.str;
       example = [ "*.example.com" "example.com" ];
     };
 
     port = mkOption {
       default = [3128];
+      type = types.int;
```

```suggestion
      type = types.listOf types.port;
```
1. looks like it needs to be a list
2. `types.port` exists and is more specific